scania-digital-design-system/sdds
# fix(tooltip): move theming rules to component folder

**Describe pull-request** 
Moving theming rules to component folder 

**Solving issue** 
Fixes: [AB#2225](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2225) 

**How to test** 
1. Open tooltip in Storybook of this branch
2. Open tooltip in Storybook of production
3. Compare styling- there should not be any difference between those two
